tests/test_vi.py

```diff
@@ -137,3 +137,6 @@ def test_special_number(self):
             num2words(1000101017, lang="vi"),
             "một tỷ một trăm lẻ một nghìn lẻ mười bảy"
         )
+
+    def test_str_to_number(self):
+        self.assertEqual(num2words("21", lang="vi"), "hai mươi mốt")
```

Comment:
@dongchirua this library is intended to transform `numbers` not `strings` into words -> strings